RELEASE_NOTES.md

```diff
@@ -4,10 +4,36 @@ Any new bugs, requests, or missing features should be reported as [Github issues
 
 ## Improvements:
 
+### Code generation
+ * Add the --short-names option when generating code, which shortens names in generated struct definitions.
```

Comment:
Since you are already mentioning generated I don't think you need to explicitly say generated structs. Just a suggestion